data/remnant/remnant 2 side missions.txt

```diff
@@ -34,7 +34,7 @@ mission "Remnant: Visit to Paradise 1"
 		conversation
 			`	You are just settling down at a table in the cafeteria when Taely slides onto the bench opposite you. "Greetings, <first>. How is life as an adventurer?"`
 			choice
-				`	"Exciting as always."`
```

Comment:
Personally, I find that this actually changes the flow of the sentence. "Exciting as always" is not the same as "Exciting, as always." I find with the comma to be a bit more sarcastic, which is not intended here.

Reply:
Interesting. I don't find the comma to make it sarcastic.
However, it is Remnant content...